### PR TITLE
develop

### PR DESCRIPTION
**Pull request by YaroslavKolbaiev**
[DEMO LINK](https://YaroslavKolbaiev.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
+          onClick={fetch5First}
+        >
+          Load 5 first goods
+        </button>
+
+        <button
+          className="button is-primary is-light"
+          type="button"
+          data-cy="red-button"
+          onClick={fetchRedGoods}
```
Comment (alexandra-protyanova) [inline src/App.tsx:61]: set red color to all red goods

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/GoodsList.tsx [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
+export const GoodsList: React.FC<Props> = ({ goods, red }) => (
   <ul className="has-text-centered">
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        className={classNames('has-text-black', {
```
Comment (TarasHoliuk) [inline src/GoodsList.tsx:16]: Unfortunately, it's still not red in the demo:

image

And that's not only about red goods:
image

Nice Bulma styles! but with those colors would be easier to just use `color` property from `good` object

File: src/GoodsList.tsx [hunk 3]
```diff
@@ -1,14 +1,22 @@
+import classNames from 'classnames';
```
Comment (TarasHoliuk) [inline src/GoodsList.tsx:1]: one of the reasons why the red color doesn't work could be the absence of `classnames` in `package.json` dependencies. It adds automatically when you run `npm install classnames`

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Good job! but you have to fix colors

**Pull request closed without merging**